From d31411677e763fe1edc3dbc2cef9b231963ae46e Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Sep 2022 12:54:15 +0000
Subject: [PATCH 1/1] Add display configuration for CR7 use case

The EVS application tries to use only displays that passed via a command
line parameters. Also the passed displays must be well-known (mentioned
in the application config file).

The CR7 hides the real HW monitors, and represent a virtual ones. As a
result the existing displays IDs aren't match anymore. So it is proposed
to add the new virtual display in the config and command line options.

Note: this patch only adds the new config for virtual display, but the
existing configuration for HW displays are redundant and can be removed.

Test: boot, check logcat and ensure that there are no related errors:
      $ adb logcat -d -v color | grep -i evs
Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
---
 config.salvator.json | 5 +++++
 evs_app.gl.rc        | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/config.salvator.json b/config.salvator.json
index e26c2ad..3bb244e 100644
--- a/config.salvator.json
+++ b/config.salvator.json
@@ -25,6 +25,11 @@
       "displayPort" : 82,
       "frontRange" : 150,
       "rearRange" : 150
+    },
+    {
+      "displayPort" : 34,
+      "frontRange" : 150,
+      "rearRange" : 150
     }
   ],
   "graphic" : {
diff --git a/evs_app.gl.rc b/evs_app.gl.rc
index 6a860d4..dacbb28 100644
--- a/evs_app.gl.rc
+++ b/evs_app.gl.rc
@@ -1,4 +1,4 @@
-service evs_app.gl /vendor/bin/evs_app.gl --hw --display 65,67,80,82
+service evs_app.gl /vendor/bin/evs_app.gl --hw --display 34,65,67,80,82
     class hal
     priority -20
     user automotive_evs
-- 
2.25.1

